### PR TITLE
Fix Reactive Spring build.gradle.kts

### PR DESCRIPTION
**Pull request by erikvanderwerf**
Current Mustache template file generates a syntax error when generating for Reactive Spring.

@dr4ke616 @karismann @Zomzog @andrewemery @4brunu @yutaka0m @stefankoppier

### PR checklist
 
- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh ./bin/configs/*.yaml
  ./bin/utils/export_docs_generators.sh
  ``` 
  (For Windows users, please run the script in [Git BASH](https://gitforwindows.org/))
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  IMPORTANT: Do **NOT** purge/delete any folders/files (e.g. tests) when regenerating the samples as manually written tests may be removed.
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master` (upcoming 7.6.0 minor release - breaking changes with fallbacks), `8.0.x` (breaking changes without fallbacks)
- [x] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

### CODE REVIEW

File: modules/openapi-generator/src/main/resources/kotlin-spring/libraries/spring-boot/buildGradle-sb3-Kts.mustache [hunk 1]
```diff
@@ -58,6 +58,6 @@ dependencies {
         exclude(module = "junit")
     }
 {{#reactive}}
-    testImplementation`("org.jetbrains.kotlinx:kotlinx-coroutines-test:$kotlinxCoroutinesVersion")
+    testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:$kotlinxCoroutinesVersion")
```
Comment (erikvanderwerf) [inline modules/openapi-generator/src/main/resources/kotlin-spring/libraries/spring-boot/buildGradle-sb3-Kts.mustache:61]: This is the primary change.

All other changes were produced by running the requested commands in Git Bash for Windows.

**Comment by wing328** [comment]
thanks for the PR.

I've filed https://github.com/OpenAPITools/openapi-generator/pull/19382 by cherry-picking your commit (only the change in template but not sample update)

**Comment by wing328** [comment]
closed via https://github.com/OpenAPITools/openapi-generator/pull/19382

**Comment by erikvanderwerf** [comment]
Thanks for merging! Did I do anything wrong when running the tools or are they out of date? @wing328

**Comment by wing328** [comment]
which version of JDK are you using? 

are you using OpenJDK or the one from Oracle?

**Comment by erikvanderwerf** [comment]
I'm using Java 20 (for no strong reason), and IntelliJ defaults to the Amazon Corretto JDK implementation.

**Pull request closed without merging**